documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -23,6 +23,28 @@ include::transaction-attributes.adoc[]
 //Doc Home
 :doc_home: https://infinispan.org/documentation/
 
+//
+// Doc Links
+//
+
+//Dev
+:dev_docs: https://access.redhat.com/documentation/en-us/red_hat_data_grid/
```

Comment:
is this link supposed to be here? is this file only for the community?

Reply:
good catch. fixed.